src/src/pages/capabilities/resources/resourceInfoBadges/index.js

```diff
@@ -472,25 +571,43 @@ export function ResourceInfoBadges() {
 
       <p style={{ textAlign: "center" }}>
         <img src={azureLogo} alt="Azure icon" style={{ height: "2.5rem" }} />
+        <Text>
+          Please refer to the{" "}
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            href="https://wiki.dfds.cloud/en/documentation/azure/azure-capability-developer"
+          >
+            Azure Capability Developer Guide
+          </a>
+        </Text>
       </p>
 
       <div className={styles.azure}>
         <div className={styles.items}>
           {azureResourcesList &&
           azureResourcesList.length !== 0 &&
           isLoadedAzure ? (
-            azureResourcesList.map((x) => (
-              <div key={x.id}>
-                <div className={styles.environment}>
-                  Azure resources for the {x.environment} environment:
-                </div>
-                <div className={styles.azureresource}>
-                  <Badge>
-                    <strong>{x.id} </strong>
-                  </Badge>
+            <>
+              <Text>
+                Your have the following existing Azure Resource Groups
```

Comment:
This reads awkwardly, though it may be correct. Can we remove s from Groups, can we put "Azure Resource Groups" in quotes? Maybe we can find a better way of phrasing it?

Reply:
Tried a new text